package-lock.json

```diff
@@ -5,12 +5,9 @@
     "requires": true,
     "packages": {
         "": {
-            "name": "cubecubed",
```

Comment:
I'm not sure about `package-lock.json` auto-generation process, but why is this line removed?